src/courtroom.cpp

```diff
@@ -2938,17 +2938,40 @@ void Courtroom::chat_tick()
 
     ui_vp_message->ensureCursorVisible();
 
-    // Blip player and real tick pos ticker
-    if (!formatting_char && (f_character != ' ' || blank_blip)) {
-      if (blip_ticker % blip_rate == 0) {
+    // We blip every "blip rate" letters.
+    // Here's an example with blank_blip being false and blip_rate being 2:
+    // I am you
+    // ! !  ! !
+    // where ! is the blip sound
+    int b_rate = blip_rate;
+    // Earrape prevention without using timers, this method is more consistent.
+    if (msg_delay != 0 && msg_delay <= 25) {
```

Comment:
What is this arbitrary number 25? Is it just `message_display_speed[2]`?

Reply:
Something like that, it's a fine-tuned number I got to obtain the least earrapey result

Reply:
Okay. Epic.